src/gui/accountstate.cpp

```diff
@@ -483,6 +487,63 @@ AccountApp* AccountState::findApp(const QString &appId) const
     return nullptr;
 }
 
+void AccountState::connectWebSocket()
+{
+    if (webSocket.state() != QAbstractSocket::UnconnectedState)
+        return;
+
+    const auto &capabilites = _account->capabilities();
+
+    // Check for web socket capability
+    // TODO: Maybe make this more finer
+    if (!capabilites.pushNotificationFilesWebSocketAvailable())
+        return;
+
+    isSupportingPushNotifications = true;
+    const auto &webSocketUrl = capabilites.pushNotificationWebSocketUrl();
```

Comment:
No ref here. Just `const auto`. Not incorrect, just unusual for no particular gain in this context.